tools/metadata.json

```diff
@@ -1,9 +1,9 @@
 {
   "StableReleaseTag": "v7.1.0",
-  "PreviewReleaseTag": "v7.2.0-preview.1",
+  "PreviewReleaseTag": "v7.2.0-preview.2",
   "ServicingReleaseTag": "v6.2.7",
```

Comment:
Is this still the right servicing release tag?

Reply:
@TravisEz13?
6.2 reached end of life on September 4, 2020.